utils/ui.js

```diff
@@ -29,6 +29,28 @@ function isElementInView(el, opt_offset) {
 }
 
 
+/**
+ * Creates a style element and inserts it into the head of the document.
+ * @param {Array.Array} A list of [<string>, <string>] elements where the first
+ *     element is the selector to create and the second element is the
+ *     corresponding style. Example: [['.foo', 'display: none']]
+ * @return {Element} Created style element.
+ */
+function createStyle(selectorsAndValues) {
+  var content = '';
+  [].forEach.call(selectorsAndValues, function(item) {
+    var selector = item[0];
+    var value = item[1];
+    content += ' ' + selector + ' { ' + value + ' }' + ' ';
```

Comment:
Not sure if you've thought about future features, but using a list for selector/value might make it difficult to add features later. For example, you might want to add support for media queries on certain values. Might be better to use a map here so it'd be easier to add stuff later on.